OPCB/functions/missions/El_Chapo.sqf

```diff
@@ -81,25 +44,18 @@ _spawncomps = [_officer] call CHAB_fnc_roadblock_ins; //double fun
 
 _hmgs = [_officer] call CHAB_fnc_spawn_hmg;
 
-with missionNamespace do
 {
 	{
-		{
-			_x addEventHandler ["FiredNear", {
-				params ["_unit", "_firer", "_distance", "_weapon", "_muzzle", "_mode", "_ammo", "_gunner"];
-				with missionNamespace do
-				{
-					Chapo_trigger = true;
-				};
-				_unit removeEventHandler ["FiredNear", _thisEventHandler];
-			}];
-		} forEach units _x;
-	} forEach enemy_groups;
-};
+		_x addEventHandler ["FiredNear", {
+			params ["_unit", "_firer", "_distance", "_weapon", "_muzzle", "_mode", "_ammo", "_gunner"];
+			ChapoTrigger = true;
+			_unit removeEventHandler ["FiredNear", _thisEventHandler];
+		}];
+	} forEach units _x;
+} forEach EnemyGroups;
 
 waitUntil {
-	_trigger = missionNamespace getVariable "Chapo_trigger";
-  !alive _officer || _trigger
+  !alive _officer || ChapoTrigger
```

Comment:
It would be good to get yourself used to using lazy evaluation with {} wherever you can

Reply:
didn't even know that was is a thing in sqf up until now xd

Reply:
I learned it very late in my scripting journey too. I used to see it in ACE code in Arma 2 and not understand it until years later xD